common/artifact-repository/overlays/abs/deployment.yaml

```diff
@@ -21,6 +21,6 @@ spec:
             - azure
           envFrom:
```

Comment:
I think it's better to set the env like this:

```
env:
  - name: MINIO_ACCESS_KEY
    valueFrom:
      secretKeyRef:
        name: onepanel
        key: artifactRepositoryS3AccessKey
  - name: MINIO_SECRET_KEY
    valueFrom:
      secretKeyRef:
        name: onepanel
        key: artifactRepositoryS3SecretKey
```

That way you don't have to do [this](https://github.com/onepanelio/cli/pull/91/files#diff-5794c40aa4c9301aaf9c39eacf7d3c8511f2511cb24bdd95e0a08704ba583a5fR372)

Reply:
@rushtehrani Ahh, good point. Much cleaner.